bundler/lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.files = IO.popen(%w[git ls-files -z], chdir: __dir__, err: IO::NULL) do |ls|
     ls.readlines("\x0", chomp: true).reject do |f|
       (f == gemspec) ||
-        f.start_with?(*%w[bin/ test/ spec/ features/ .git <%= config[:ci_config_path] %>appveyor Gemfile])
+        f.start_with?(*%w[bin/ test/ spec/ features/ .git <%= config[:ci_config_path] %>appveyor <%= config[:linter_config_path] %>Gemfile])
```

Comment:
Not something for this PR at all, but just noticed that `appveyor` feels pretty random here, since it's not a supported CI provider in `bundle gem`. I'd either completely drop it, or add it as a supported CI provider for `bundle gem`.